docs/core-concepts/model-basics.md

```diff
@@ -278,6 +278,35 @@ Foo.init({ /* attributes */ }, {
 });
 ```
 
+## Prevent creating a default PK attribute
+
+By default, Sequelize automatically adds the primary key field `id` to every model when there is no primary key field defined. When there is an `id` field but not a primary key field Sequelize will throw an error. To prevent this you can use the option `noPrimaryKey` when creating the instance and set it to true.
+
+```js
+const sequelize = Sequelize.createSequelizeInstance({
```

Comment:
`Sequelize.createSequelizeInstance` does not exist, it's a helper we use in our tests.

The way to create a sequelize instance is just `new Sequelize`

Reply:
Woops, I missed that. Good one!